pkg/util/policy/agent/agent.py

```diff
@@ -56,6 +47,9 @@ def filter_rules_idx(rules):
 # We want to receive json object as a POST.
 class AgentHandler(BaseHTTPRequestHandler):
     def do_GET(self):
+        """
+        We do not have a storage so GET is not really supported.
```

Comment:
I suppose we could examine what's stored in iptables, extract the policy-IDs (or names?) that were stored there (in the comments) and return at least a list of the polict IDs/names, even if it's not the full policy definition.

This might be useful for testing even, since we could have a black-box test that can double check that certain policies have been applied to iptables.